cmake/modules/hunter_dump_cmake_flags.cmake

```diff
@@ -20,6 +20,25 @@ function(hunter_dump_cmake_flags)
     hunter_internal_error("Unparsed arguments: ${x_UNPARSED_ARGUMENTS}")
   endif()
 
+
```

Comment:
I'm not sure about odb-boost only.
hunter_dump_cmake_flags is also used in hunter_autotools_project, that is used in:

- Avahi
- SQLite3
- odb-sqlite
- xcb
- odd-pgsql
- libsodium

should I update PACKAGE_INTERNAL_DEPS_ID for all those packages as well?